api/sdkmeta/data/languages.json

```diff
@@ -0,0 +1,108 @@
+{
```

Comment:
Due to limitations of the `embed` module in Go, the embedded files need to be in a child directory of the module. So I've updated the shell scripts to do a `cp -r` and duplicate the data here.

Not optimal but gets the job done.